### PR TITLE
Remove stale information from benchmark test README files

### PR DESCRIPTION
**Pull request by peter-mitsis**
Remove stale information from benchmark test README files.

The app_kernel project had ...
   1. A stale reference to the microkernel (which has been gone for years)
   2. Obsolete build instructions
   3. Sample output that contained defunct functionality

The sys_kernel project had ...
   1. Obsolete build instructions
   2. References to TICKS_NONE instead of K_NO_WAIT
   3. Some miscellaneous stale output

### CODE REVIEW

File: tests/benchmarks/app_kernel/README.txt [hunk 1]
```diff
@@ -1,31 +1,9 @@
...
-
-Problems caused by out-dated project information can be addressed by
-issuing one of the following commands then rebuilding the project:
-
-    make clean          # discard results of previous builds
-                        # but keep existing configuration info
-or
-    make pristine       # discard results of previous builds
-                        # and restore pre-defined configuration info
+The app_kernel project is used to measure the performance of the following
```
Comment (kartben) [inline tests/benchmarks/app_kernel/README.txt:5]: Nit


```suggestion
The app_kernel test is used to measure the performance of the following
```

**Review by kartben — DISMISSED** [review]
Thanks!

**Review by nashif — DISMISSED** [review]

File: tests/benchmarks/app_kernel/README.txt (hunk 1 above)
Comment (peter-mitsis) [reply tests/benchmarks/app_kernel/README.txt]: Fixed.

**Comment by peter-mitsis** [comment]
@kartben and @nashif - Very minor tweak--as per Benjamin's comment, I changed "app_kernel project" to "app_kernel test".

**Review by kartben — APPROVED** [review]

**Review by nashif — APPROVED** [review]

**Review by fabiobaltieri — APPROVED** [review]

**Review by dcpleung — APPROVED** [review]

**Review by jhedberg — APPROVED** [review]

**Pull request merged**